docs/MockFunctionAPI.md

```diff
@@ -49,6 +49,24 @@ that looks like this:
 ['result1', 'result2'];
 ```
 
+NOTE: For any call to the mock that throws an error, a value of `undefined` will
+be stored in `mock.returnValues`.
```

Comment:
Could consider including this in the example rather than a note:

> For example: A mock function `f` that has been called three times, returning
 `result1`, then returning `result2`, and then throwing an error would have a `mock.returnValues` array
 that looks like this:	 that looks like this:

> ['result1', 'result2', undefined];

Reply:
I updated the documentation.